subsys/net/lib/coap/Kconfig

```diff
@@ -76,6 +76,13 @@ config COAP_URI_WILDCARD
 	  This option enables MQTT-style wildcards in path. Disable it if
 	  resource path may contain plus or hash symbol.
 
+config COAP_KEEP_SOCKET
+	bool "Enable keeping socket fd for sending reply"
+	default n
```

Comment:
No need to set `default n` explicitly.

Reply:
@rlubos CoAP library currently has no handlers to add app-specific data into CoAP message.